### DISCUSSION TITLE
Anything significant missing with this plugin?

### DISCUSSION THREAD

**Discussion by garyniemen** [drodil/backstage-plugin-qeta#148] [General]
This plugin looks to have all the basics needed for a question and answer tool. Do any existing users feel it is missing anything significant - as compared, say, with Stack Overflow?

**Comment by drodil** [comment]
Hey, thanks for the question! I think one of the major things missing is notifications support which is coming when the Backstage upstream notification work is done. It will help users to get notified when their questions are answered or if a new question is asked about their entity. But it should be doable soonish.

**Reply by garyniemen** [reply]
Do you know if they are working on that?

**Reply by drodil** [reply]
I am part of the people implementing it and it seems 1.26.0 was shipped yesterday so I could start integrating notifications to this plugin. Hopefully I find time this week!

**Reply by garyniemen** [reply]
Wow, that answer I did not expect. Fantastic.

**Reply by drodil** [reply]
Actually, just created the first implementation for notifications. Will publish a new version of the plugin for people to try it out. The notifications support requires some extra plugins to be installed but there's instructions what to do. I will install this on my own Backstage instance tomorrow and try it out with bigger audience :)

**Reply by garyniemen** [reply]
I was part of the Backstage team when I was at Spotify and we talked about having some kind of Notifications functionality so many times - but it never happened. So just to be clear - the new version of Backstage has a notifications framework, right?

**Reply by drodil** [reply]
Yep, it was initially already in 1.24.0, but in early alpha. It's still under development but 1.26.0 it's already usable. Of course there are some things missing still and they are more closely discussed in the BEP: https://github.com/backstage/backstage/tree/master/beps/0001-notifications-system

**Reply by drodil** [reply]
@garyniemen and pinging here too, this plugin now supports Backstage notifications natively :)
